server/src/services/organization_constants.service.ts

```diff
@@ -0,0 +1,184 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { OrganizationConstant } from '../entities/organization_constants.entity';
+import { dbTransactionWrap } from 'src/helpers/utils.helper';
+import { EncryptionService } from './encryption.service';
+import { AppEnvironmentService } from './app_environments.service';
+
+import { DeleteResult, EntityManager, Repository } from 'typeorm';
+import { CreateOrganizationConstantDto, UpdateOrganizationConstantDto } from '@dto/organization-constant.dto';
+
+@Injectable()
+export class OrganizationConstantsService {
+  constructor(
+    @InjectRepository(OrganizationConstant)
+    private organizationConstantsRepository: Repository<OrganizationConstant>,
+    private encryptionService: EncryptionService,
+    private appEnvironmentService: AppEnvironmentService
+  ) {}
+
+  async allEnvironmentConstants(organizationId: string): Promise<OrganizationConstant[]> {
+    return await dbTransactionWrap(async (manager: EntityManager) => {
+      const query = manager
+        .createQueryBuilder(OrganizationConstant, 'organization_constants')
+        .leftJoinAndSelect('organization_constants.orgEnvironmentConstantValues', 'org_environment_constant_values')
+        .where('organization_constants.organization_id = :organizationId', { organizationId });
+      const result = await query.getMany();
+
+      const appEnvironments = await this.appEnvironmentService.getAll(organizationId, null);
```

Comment:
```suggestion
      const appEnvironments = await this.appEnvironmentService.getAll(organizationId);
```

Reply:
since the function accepts manager, and manager is available here, we can pass manager obj
```
const appEnvironments = await this.appEnvironmentService.getAll(organizationId, manager);
```